classes/01class/exercises/c01-git02/carloshz4/README.md

```diff
@@ -0,0 +1,97 @@
+# c01-git02
+
+## Answers
+
+### 1. Is it possible to see the inner commit of squashed commit with `git rebase`?
+Yes, using:
+
+```
+git reflog
+```
+For instance getting this output:
+
+```
+46f3319 HEAD@{0}: rebase -i (finish): returning to refs/heads/master
+46f3319 HEAD@{1}: rebase -i (squash): Squashed commit with Add ANSWER2.md again
+abc6d70 HEAD@{2}: rebase -i (squash): # This is a combination of 9 commits.
+fbf04aa HEAD@{3}: rebase -i (squash): # This is a combination of 8 commits.
+63bd91c HEAD@{4}: rebase -i (squash): # This is a combination of 7 commits.
+384a7ad HEAD@{5}: rebase -i (squash): # This is a combination of 6 commits.
+dfb9768 HEAD@{6}: rebase -i (squash): # This is a combination of 5 commits.
+c920c22 HEAD@{7}: rebase -i (squash): # This is a combination of 4 commits.
+c2d5bb5 HEAD@{8}: rebase -i (squash): # This is a combination of 3 commits.
+77e46ee HEAD@{9}: rebase -i (squash): # This is a combination of 2 commits.
+643c725 HEAD@{10}: checkout: moving from master to 643c725
+aec3935 HEAD@{11}: commit: latest one buddy
+91f4663 HEAD@{12}: commit: add test readme
+f773185 HEAD@{13}: commit: add test readme
```

Comment:
and it is also possible to checkout commits, even from the remote repo

Reply:
Thank you Denis for the comment and suggestion.